app/views/meow/products/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>All Cats</h1>
+
+<% @products.each do |product| %>
+
+<%= product.breed %>
+<%= product.dob %>
+<%= product.description %>
+<%= product.price %>
```

Comment:
Oh yeah we need to either add the $ or SGD here in the view. And format the price to 2 decimal places.